tests/fixtures/nwchem/references.py

```diff
@@ -0,0 +1,67 @@
+# Reference energies are in units of eV.
+# NWChem returns energies in hartrees, ExPrESS converts to eV
+# Reference energies are all in eV
+TOTAL_ENERGY = -2079.18666382721904
+
+TOTAL_ENERGY_CONTRIBUTION = {
+    "one_electron": {
+        "name": "one_electron",
+        "value": -3350.531714067630674
+    },
+    "coulomb": {
+        "name": "coulomb",
+        "value": 1275.68347728573713
+    },
+    "exchange_correlation": {
+        "name": "exchange_correlation",
+        "value": -254.54658374762781
+    },
+    "nuclear_repulsion": {
+        "name": "nuclear_repulsion",
+        "value": 250.20815670232923
+    }
+}
+
+BASIS = {
+    "units": "angstrom",
```

Comment:
Are we able to retain the fact that this is an isolated molecule, rather than a periodic system, using some of the latest addition(s) to ESSE?

Reply:
we have the following description in `esse/schema/material.json`
``` 
"isNonPeriodic": {
            "description": "Whether to work in the finite molecular picture (usually with atomic orbital basis)",
            "type": "boolean"
        }
```

Reply:
Yep, this sounds like the right direction. Did we adjust the Basis itself as well?

Reply:
No i don't think we adjusted the basis yet. It just shows coordinates, bonds, and elements right now.
```
    "properties": {
        "elements": {
            "$ref": "basis/atomic_elements.json"
        },
        "coordinates": {
            "$ref": "basis/atomic_coordinates.json"
        },
        "bonds": {
            "$ref": "basis/bonds.json"
        }
    },
    "required": [
        "elements",
        "coordinates"
    ]
}
```

Reply:
That's what I mean: we added [bonds](https://github.com/Exabyte-io/esse/pull/149/files#diff-198643158ad121836826a8d45a0418706b4c67f4a7e0866b77ea4b51688e4431R2) data preparing for storing the data. Does NWChem produce any useful data that we can retain in the `bonds` object? Alternatively, we would need to populate the bond data separately after the fact.

Reply:
We could add in the stretch and bend parameters for the molecule into the bond data.

Reply:
```
      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95720
    2 Stretch                  1     3                       0.95720
    3 Bend                     2     1     3               104.52019
```